client/src/views/ActiveUsersPage.vue

```diff
@@ -0,0 +1,183 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS -->
+
+<template>
+  <ion-page>
+    <ion-content :fullscreen="true">
+      <!-- contextual menu -->
+      <ion-item-divider class="users-toolbar ion-margin-bottom secondary">
+        <button-option
+          id="button-invite-user"
+          :button-label="$t('UsersPage.inviteUser')"
+          @click="inviteUser()"
+          v-show="isAdmin()"
+        />
+        <!-- <div class="right-side">
+          <ms-select
+            id="user-filter-select"
+            :options="msSelectOptions"
+            default-option="name"
+            @change="onMsSelectChange($event)"
+          />
+          <list-grid-toggle
+            :list-view="false"
+            @toggle-view="onToggleView($event)"
+          />
+        </div> -->
```

Comment:
why is all this code commented?
Do we want to keep it?

Reply:
Forgot about it. I removed the sort part and kept the list/grid toggle (updated it to use v-model).